gradle.properties

```diff
@@ -0,0 +1,2 @@
+group=com.github.twitch4j
+version=1.2.0-SNAPSHOT
```

Comment:
Almost forgot this. `groupId` and `version` moved to this file. [This plugin](https://github.com/KengoTODA/gradle-semantic-release-plugin/) works better if version will be here.